.github/workflows/release.yml

```diff
@@ -36,25 +34,37 @@ jobs:
           node-version: 18
 
       - run: npm install
-
       - run: npm run build
-
       - run: npm run test
         if: ${{ inputs.skip-test == false }}
+      - name: Upload release artifacts
+        id: upload-release
+        uses: actions/upload-artifact@v4
+        with:
+          name: release
+          path: lib
 
       - name: Configure AWS Credentials
-        uses: aws-actions/configure-aws-credentials@v2
+        uses: aws-actions/configure-aws-credentials@v4
         with:
           role-to-assume: ${{ secrets.AWS_CODEARTIFACT_ROLE }}
```

Comment:
Is this code still needed? If yes, I guess we stick with "codeartifact role" as legacy, but ideally should fix this too

Reply:
Yes this is configuring the credentials to allow access to codebuild. But can rename the role.